### PR TITLE
Group prose additions

### PR DESCRIPTION
**Pull request by jeromefv**
# Additions
- After PR #5492 
- Introduction paragraphs for SDK groups
    - Core, Components, Editors, Binder, Jupiter
- [ ] Mythic information to complete
- [ ] Binder example explanation

- [x] I have read the [Contributor Guide](https://github.com/nteract/nteract/blob/master/CONTRIBUTING.md)
- [x] I have updated the changelogs/current_changelog.md file with some information about the change that I am making the appropriate file.
- [x] I have validated or unit-tested the changes that I have made.
- [x] I have run through the TEST_PLAN.md to ensure that my change does not break anything else.

### CODE REVIEW

File: docs/groups/binder-group.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Binder
 
+The **Binder** group contains the SDK packages for working with [Binder](https://mybinder.org) specifications. This group supports users and Binder APIs providing features such as connecting to notebooks, notebook servers, as well as notebook instances.
```
Comment (captainsafia) [inline docs/groups/binder-group.md]:
```suggestion
The **Binder** group contains the SDK packages for working with [Binder](https://mybinder.org) specifications. This group allows developers to leverage Binder APIs to provide features such as connecting to notebooks, notebook servers, as well as notebook instances.
```

File: docs/groups/components-group.md [hunk 2]
```diff
@@ -1,5 +1,7 @@
 # Components
 
+The **Components** group is the set of SDK packages for displaying and styling notebooks. This group provides the tools for manipulating visual notebook elements.
```
Comment (captainsafia) [inline docs/groups/components-group.md:3]:
```suggestion
The **Components** group is the set of SDK packages for displaying and styling notebooks. This group provides the tools for manipulating visual notebook elements, like cell inputs and outputs.
```

**Review by captainsafia — COMMENTED** [review]
"The mythic group contains an abstraction for writing individual pieces of business logic into packages, referred to as myths, that contain definitions for Redux actions, reducersm and epics."

File: docs/groups/components-group.md (hunk 2 above)
Comment (willingc) [reply docs/groups/components-group.md:3]: Going to commit this suggestion to see if the new ci will kick in.

File: docs/groups/mythic-group.md [hunk 3]
```diff
@@ -1,5 +1,7 @@
 # Mythic
 
+The Mythic group contains an abstraction for writing individual pieces of business logic into packages. These packages are Myths and contain definitions for Redux actions, reducersm and epics.
```
Comment (captainsafia) [inline docs/groups/mythic-group.md]:
```suggestion
The Mythic group contains an abstraction for writing individual pieces of business logic into packages. These packages are Myths and contain definitions for Redux actions, reducers, and epics.
```

File: docs/groups/binder-group.md [hunk 4]
```diff
@@ -1,5 +1,7 @@
 # Binder
 
+The **Binder** group contains the SDK packages for working with [Binder](https://mybinder.org) specifications. This group allows developers to leverage Binder APIs to provide features such as connecting to notebooks, notebook servers, as well as notebook instances.
+
 **Table of contents**
 
 [TOC]
```
Comment (captainsafia) [inline docs/groups/binder-group.md:7]: Do we have a plan for what to do about this ToC headers?

Comment (jeromefv) [reply docs/groups/binder-group.md:7]: The `[TOC]` text shows 2-3 headers deep of the layout and comes from the `markdown_extensions` in the main mkdocs.yml file. It builds as a normal ToC for the live page.

**Comment by captainsafia** [comment]
@jeromefv Is this ready to take out of draft mode?

**Comment by jeromefv** [comment]
@captainsafia Just changed text from tentative answer to something like an "under construction" phrase. Took it off review, so it's ready. Thanks!

**Review by willingc — APPROVED** [review]
Awesome!

**Pull request merged**